test/getOldestInstance.test.ts

```diff
@@ -0,0 +1,18 @@
+import {findOldestInstance, Instance} from '../src/getOldestInstance'
```

Comment:
If these are only being exported from the module for the purpose of testing, we might like to borrow the pattern from frontend where an object named `_` is exported

In module:
```
export const _ = { findOldestInstance, Instance };
```

In test file:

```
import { _ } from '../src/getOldestInstance'

const { findOldestInstance, Instance } = _
```

More a suggestion than a blocker 😁 Might be something we decide to do later

Reply:
Thanks, I was wondering if there was a neater way to do this for testing purposes only 👍